### DISCUSSION TITLE
How to clear OutputWindowPane

### DISCUSSION THREAD

**Discussion by Denisjc** [VsixCommunity/Community.VisualStudio.Toolkit#483] [Q&A] [answered]
How do we clear the OutputWindowPane  once we have logged to it?

**Comment by RobertvanderHulst** [comment, marked as answer]
> How do we clear the OutputWindowPane once we have logged to it?

You don't. The user should be in control.

**Comment by reduckted** [comment]
https://github.com/VsixCommunity/Community.VisualStudio.Toolkit/blob/4fa5cfc95253ac2b40d3ea8c52e14b106eabade2/src/toolkit/Community.VisualStudio.Toolkit.Shared/Helpers/OutputWindowPane.cs#L202
